test/cljs/status_im/test/chat/commands/impl/transactions.cljs

```diff
@@ -38,11 +38,7 @@
            {:title       (i18n/label :t/send-request-amount)
             :description (i18n/label :t/send-request-amount-max-decimals {:asset-decimals 18})}))
     (is (= (protocol/validate personal-send-command {:asset "ETH" :amount "0.01"} cofx)
-           nil)))
-  (testing "Yielding control prefills wallet"
```

Comment:
Any reason to remove that?

Reply:
yes, the side-effect it's testing no longer exists, as `:wallet :send-transaction` in `app-db` is no longer used, so there's nothing to check now.